tests/foreman/ui/test_syncplan.py

```diff
@@ -171,7 +171,7 @@ def test_positive_end_to_end_custom_cron(session):
         assert not syncplan_values['details']['cron_expression']
         # Delete sync plan
         session.syncplan.delete(plan_name)
-        assert not session.syncplan.search(plan_name)
+        assert 'Your search returned zero Sync Plans.' in str(session.syncplan.search(plan_name))
```

Comment:
Why every time you are converting to str object?

Reply:
The is one assert at the end of every test. Maybe I misunderstand?

Reply:
I was talking about `str(session.syncplan.search(plan_name))`. why there explicitly covert object into a string.   

if this not works 
`assert 'Your search returned zero Sync Plans.' in session.syncplan.search(plan_name)`

try below ones ?

`assert 'Your search returned zero Sync Plans.' in session.syncplan.search(plan_name)[0]['Name']`
or 
`session.syncplan.search(plan_name)[0]['Name'] != plan_name`

Reply:
OK, I did try some of those before, trying again with 1:

    >           assert empty_search_message in session.syncplan.search(plan_name)
    E           AssertionError: assert 'Your search returned zero Sync Plans.' in [{'Description': 'Your search returned zero Sync Plans.', 'Interval': 'Your search returned zero Sync Plans.','Name': 'Your search returned zero Sync Plans.', 'Next Sync': 'Your search returned zero Sync Plans.', ...}]

2nd one works, and 3rd one works:
    assert session.syncplan.search(plan_name)[0]['Name'] != plan_name

But there is this to consider:
https://github.com/SatelliteQE/robottelo/issues/7272

Reply:
This method works, and complies with https://github.com/SatelliteQE/robottelo/issues/7272


    @@ -173,7 +173,7 @@ def test_positive_end_to_end_custom_cron(session):
         assert not syncplan_values['details']['cron_expression']
         # Delete sync plan
         session.syncplan.delete(plan_name)
    -        assert empty_search_message in str(session.syncplan.search(plan_name))
    +        assert plan_name not in session.syncplan.search(plan_name)
 
 
     @tier2
    @@ -209,8 +209,7 @@ def test_positive_search_scoped(session):
         ]:
             assert session.syncplan.search(
                 '{} = {}'.format(query_type, query_value))[0]['Name'] == name
    -        assert empty_search_message in str(
    -            session.syncplan.search('enabled = false'))
    +        assert name not in session.syncplan.search('enabled = false')

Reply:
Changes made, will change pull request title to match latest change.

Reply:
I could run a test with "not" removed from those asserts just to make sure it fails.

Reply:
I did that, removing the "not" makes the assert fail.